tests/src/Functional/ModalPageBootstrapLibraryTest.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\oe_whitelabel\Functional;
+
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Tests that Bootstrap libraries are removed when oe_whitelabel is active.
+ */
+class ModalPageBootstrapLibraryTest extends BrowserTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'oe_whitelabel_helper',
+    'oe_whitelabel_modal_page',
```

Comment:
We don't need to enable the module for this test, do we?
```suggestion
    'modal_page',
    'oe_whitelabel_helper',
```

Reply:
True, now it's enough to enable modal_page. Fixed it in [360baf6](https://github.com/openeuropa/oe_whitelabel/pull/390/commits/360baf600e9da6b426bf3c6d1632f32c591751d2).